docs/source/changelog/bugfix/add_shape_to_tuple.rst

```diff
@@ -0,0 +1,7 @@
+[Bugfix] Addition between Shape and tuple
+===========================================================
+
+* Tuples can be added directly to Shape objects when either
+ the signal or navigation dimensions of the Shape object are zero
```

Comment:
FYI I think the CI is complaining about this line: the second line of the first bullet point needs to be indentation-aligned to the beginning of the first line ("the" directly below "Tuples")

Reply:
Oh I ignored Travis because I thought it was still broken. I fixed it.